### PR TITLE
CC assorted updates (FAQs, IP whitelist)

### LINKED ISSUE #4677: CC: Update backup FAQ
From Slack: "We keep full backups for 30 days and incremental backups for 7 days"

### PR DESCRIPTION
**Pull request by Amruta-Ranade**
Closes #4677 #5594 #4948

### CODE REVIEW

**Comment by Amruta-Ranade** [comment]
cc @jeremycbloom

**Comment by kannanlakshmi** [comment]
lgtm, thanks!

File: netlify/serve [hunk 1]
```diff
@@ -1,10 +1,10 @@
-#!/bin/bash
+#bin/bash
```
Comment (chrisseto) [inline netlify/serve]: Why was the `!/` removed? `#!` is required for [shebangs](https://en.wikipedia.org/wiki/Shebang_(Unix))

Comment (Amruta-Ranade) [reply netlify/serve]: Oops..didn't mean to do that. Thanks for catching it! Fixed.

**Comment by kannanlakshmi** [comment]
Nice LGTM! I like the updated "how do I connect to my cluster" question in particular, its very clear!

**Comment by kannanlakshmi** [comment]
:lgtm:

**Pull request merged**